src/api.rs

```diff
@@ -197,7 +197,7 @@ pub struct AuthenticatedApi<'a> {
 pub struct RegionSpecificApi<'a, 'b> {
     api: &'a AuthenticatedApi<'a>,
     org: &'b str,
```

Comment:
I'm starting to wonder if it'd be better to just clone these strings to avoid potential lifetime issues in the future? I suppose this only matters if there's a situation where the API outlives the org or region_url references which seems unlikely.

Reply:
Right now, we simply create the region specific API on the fly when we need it to make a request. We don't store it around, so it will always be shorter-lived than both the `api` and the `org` references